acceptance-tests/tests/helpers/cmd.js

```diff
@@ -127,7 +130,10 @@ function executeWithInput(processPath, args = [], inputs = [], opts = {}) {
         clearTimeout(currentInputTimeout);
         inputs = [];
       }
-      reject(err.toString());
+
+      // For some reason, when we use 'ora', it is throwing this error,
+      // so for now we won't reject.
+      console.error(err.toString());
```

Comment:
Does this have an impact on any other tests for commands that aren't using ora?

Reply:
Not that I could tell, I'll do some extra testing

Reply:
Ok from what I could tell everything still works great

Reply:
I ended up changing it to use a blacklisted string approach, just in case.  More granular that way